src/app/components/OptimoPromos/Image/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { string, number } from 'prop-types';
+import ImageWithPlaceholder from '#app/containers/ImageWithPlaceholder';
+import StyledImage from './index.styles';
+
+const Image = ({ src, altText, srcset, fallbackSrcset, width, height }) => {
+  const ASPECT_RATIO = (9 / 16) * 100;
+
+  return (
+    <ImageWithPlaceholder
+      src={src}
+      alt={altText}
+      srcset={srcset}
+      fallbackSrcset={fallbackSrcset}
+      ratio={ASPECT_RATIO}
+      width={width}
+      height={height}
+      lazyload
+      imageComponent={StyledImage}
+    />
+  );
+};
+
+Image.propTypes = {
+  src: string.isRequired,
+  altText: string.isRequired,
+  width: number.isRequired,
+  height: number.isRequired,
+  srcset: string,
+  fallbackSrcset: string,
+};
+
+Image.defaultProps = { srcset: '', fallbackSrcset: '' };
```

Comment:
Are these default props needed, I imagine the componet doesn't work with empty values for these props?